.github/workflows/test.yml

```diff
@@ -0,0 +1,26 @@
+name: 'Test'
```

Comment:
Added a test workflow as part of this. ref: https://app.asana.com/0/1100423001970639/1201381902617444/f